components/Main.vue

```diff
@@ -1,8 +1,274 @@
 <template>
-  <v-row>
-    <v-col class="text-center text-h3">
-      Let's go!
-      <v-icon color="success" icon="mdi-rocket-launch-outline" size="x-small" />
+  <v-row :class="[$vuetify.theme.dark ? 'currency-converter dark-theme' : 'currency-converter light-theme']">
```

Comment:
On peut le simplifier :

```suggestion
  <v-row :class="['currency-converter', $vuetify.theme.dark ? 'dark-theme' : 'light-theme']">
```

Reply:
C'est bien d'avoir identifié que Vuetify gère le thème sombre. Vuetify gère des thèmes complets, compatible avec tous les composants (juste une config et paf ça change toutes les couleurs d'un coup).
Mais c'est un peu trop de boulot dans le cadre de cet exercice.

Reply:
J 'ai modifié ça merci.